openbb_terminal/stocks/options/options_controller.py

```diff
@@ -1542,3 +1588,30 @@ def call_hedge(self, _):
     def call_screen(self, _):
         """Process screen command"""
         self.queue = screener_controller.ScreenerController(self.queue).menu()
+
+    @log_start_end(log=logger)
+    def call_eodchain(self, other_args: List[str]):
+        """Process eodchain command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="eodchain",
+            description="Get full option chain at a given end of day.",
+        )
+        parser.add_argument(
```

Comment:
Should be required, otherwise we'll get `Error: descriptor 'strftime' for 'datetime.date' objects doesn't apply to a 'NoneType' object`